_data/projects/validator/releases/8.0/series.yml

```diff
@@ -9,7 +9,7 @@ maven:
       summary: Annotation processor
 integration_constraints:
   java:
-    version: 11, 17 or 21
+    version: 11, 17, 21 or 22
```

Comment:
I see that 8.0 is tested against all up to 24 so should we mention 23 here or ...
https://github.com/hibernate/hibernate-validator/blob/b23a4c481b783994d18388830cd6f7bbfcd93421/Jenkinsfile#L127-L138

Reply:
Will do, thanks.